docs/category/category.yaml

```diff
@@ -20,19 +36,27 @@ paths:
                 - _id: 64045f98b131dd04d7896af6
                   name: languages
                   categoryIcon: categories-icons/link-to-icon.png
-                  subjectsIcon: subjects-icons/link-to-icon.png
                   totalOffers: 9
                   subjects:
-                    $ref: '#/definitions/subjects'
+                    - _id: 640f729d3c34d04e8980916e
+                      name: English
+                      category: 64085af5a129028f21309378
+                      totalOffers: 9
+                      createdAt: 2023-03-13T18:59:41.671Z
```

Comment:
think what fields to return, do we really need to return createAt and updateAt for category?

Reply:
In future there will be an ability to manage categories, so I think we may need them